app/controllers/application_controller.rb

```diff
@@ -29,6 +31,14 @@ class ApplicationController < ActionController::Base
     end
   end
 
+  def show_max_size_warning?
+    ENV['SHOW_LOOMIO_ORG_MARKETING'] and @group and @group.approaching_max_size? and !@group.max_size_reached?
```

Comment:
Don't use `and`, use &&. This will error if `@group` is nil, and will evaluate all of these possibilities each time.